app/managers.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self._cursor = self._connection.cursor()
+
+    def create(self, first_name: str, last_name: str) -> Actor:
+        self._connection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:

```

Comment:
The `all` method should also include error handling to manage potential database access issues.